internal/job/executor.go

```diff
@@ -100,7 +100,7 @@ func (e *Executor) Run(ctx context.Context) (exitCode int) {
 	defer stopper()
 	defer func() { span.FinishWithError(err) }()
 
-	// Create a context to use for cancelation of the job
+	// Create a context to use for cancellation of the job
 	cancelCtx, cancel := context.WithCancel(ctx)
```

Comment:
given that this is now the default context for the executor, i think we should just call it `ctx` - it encourages people to use this one as it seems like the default